src/build/buildNativefierApp.ts

```diff
@@ -123,9 +139,10 @@ export async function buildNativefierApp(
   const appPathArray = await electronPackager(options.packager);
 
   log.info('\nFinalizing build...');
-  const appPath = getAppPath(appPathArray);
+  const appPaths =
+    options.packager.arch === 'all' ? appPathArray : [getAppPath(appPathArray)];
 
-  if (appPath) {
+  if (appPaths) {
```

Comment:
Sorry, my python background snuck in there. :)

Reply:
> Sorry, my python background snuck in there. :)

Hehe, I can relate 😄. Sadly, we'll never see an empty array being trueish, because backwards compatibility. But thankfully, new JS features are now better designed, with proper RFCing. Also, I WANT LIST/GENERATOR COMPREHENSIONS IN JAVASCRIPT.

Reply:
> I WANT LIST/GENERATOR COMPREHENSIONS IN JAVASCRIPT.

Amen.